src/common/json_request.cpp

```
@@ -122,9 +114,9 @@ std::string EncodeBase64Safe(const std::vector<uint8_t>& data) {
 // JsonRequest implementation
 void JsonRequest::ParseCommon(const std::string& request_body) {
     // Load and validate JSON first
-    auto json_body = crow::json::load(request_body);
-    
-    if (!json_body) return; // Stop parsing if JSON is invalid
+    auto json_body_opt = SafeLoadJsonBody(request_body);
+    if (!json_body_opt) return; // Stop parsing if JSON is invalid
```

silent failure? (asking, I'm not sure)